apps/addons/decorators.py

```diff
@@ -14,9 +14,12 @@ def addon_view(f, qs=Addon.objects.all):
     def wrapper(request, addon_id=None, app_slug=None, *args, **kw):
         """Provides an addon given either an addon id or an addon slug."""
         assert addon_id, 'Must provide addon id or slug'
-        get = lambda **kw: get_object_or_404(qs(), **kw)
+
+        def get_or_404(**kw):
```

Comment:
Is this function actually useful ? I'd just call `get_object_or_404(qs(), ...)` in both cases and get rid of it.